.buildkite/pipeline.yml

```diff
@@ -31,6 +31,7 @@ steps:
       - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
       - 'ln -s /usr/local/bin/firecracker testdata/firecracker'
       - 'ln -s /usr/local/bin/jailer testdata/jailer'
-      - "make test EXTRAGOARGS='-v -count=1'"
+      - 'sudo chown 999:999 testdata/vmlinux'
```

Comment:
Do you need to `chown` it?  999 is not guaranteed to be the UID of the current user.

Reply:
Was hard coding for testing. I managed to move it into code. However, seems to be another issue with the tests.

Reply:
Want to pair-program on this tomorrow?  Let's get this resolved and merged!